src/rust/engine/pantsd/build.rs

```diff
@@ -0,0 +1,6 @@
+use std::process::Command;
+fn main() {
+    let output = Command::new("git").args(&["describe", "--tags"]).output().unwrap();
+    let git_tag = String::from_utf8(output.stdout).unwrap();
+    println!("cargo:rustc-env=GIT_TAG={}", git_tag);
+}
```

Comment:
I don't think this'll work without some kind of `rerun-if`, because this crate could just be cached.

I don't have any good ideas though...

Reply:
Doing this with https://stackoverflow.com/questions/43753491/include-git-commit-hash-as-string-into-rust-program would allow for doing it in our `cargo` wrapper scripts, so it would always re-run, but usually be identical, afaik.

Reply:
That is what I'm doing here, no?!

Reply:
Ah, well this is solved by moving the `git describe` to `./cargo` and just reading the env in the `lib.rs`.

Closing this :tada: